scss/components/_banner.scss

```diff
@@ -1,6 +1,18 @@
 .banner {
-    background-color: $astral;
-    color: $white;
+    background-color: $white;
+
+    &--prototype {
```

Comment:
Do we want to use a more generic name than prototype? Could it get used for other reasons?